modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js

```diff
@@ -1357,6 +1363,9 @@
 				this.addStartTimeCheck();
 				this.play();
 			}
+		},
+		setInline: function ( state ) {
+			_this.getPlayerElement().attr('webkit-playsinline', '');
```

Comment:
@nissopa did you check this code - are you sure its working? - please recheck this line...